index.html

```diff
@@ -24,14 +24,167 @@ <h2 class="intro-title">Hey there. I'm Juan Carlos</h2>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
```

Comment:
[ Required ] Please consider changing the all the project cards ```<div="class-container"> ``` to use the semantic `section` tag instead of the generic `div` tag. Also change the `p` tags in the project cards to use a heading tag such as `h3`, `h4`, `h5`, or `h6` these two changes will make it easier for screen readers to read through your web page accurately.

## Example
 ```HTML
           <section class="card-container">
                    <div class="card-info">
                        <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
                        <ul class="skills-container">
                            <li class="work-skill">Ruby on rails</li>
                            <li class="work-skill">css</li>
                            <li class="work-skill">JavaScript</li>
                            <li class="work-skill">html</li>
                        </ul>
                        <button type="button" class="default-button">See Project</button>
                    </div>
            </section>```